grails-app/views/profile/_lists.gsp

```diff
@@ -21,16 +21,37 @@
         <a name="{{listCtrl.readonly ? 'view_' : 'edit_'}}conservationStatus"></a>
 
         <div class="panel-body">
-            <div class="col-sm-2"><strong>Conservation status</strong></div>
-
-            <div class="col-sm-10">
-                <div class="col-sm-4" ng-repeat="status in listCtrl.conservationStatuses">
-                    <a href="${grailsApplication.config.collectory.base.url}/public/showDataResource/{{listCtrl.statusRegions[status.region].id}}"
-                       title="Threatened Species Codes - details" target="_blank">
-                        <div class="status"
-                             ng-class="listCtrl.getColourForStatus(status.status)">{{listCtrl.statusRegions[status.region].abbrev | default:'IUCN'}}</div>
-                        {{status.rawStatus}}
-                    </a>
+            <div class="row">
+                <div class="col-sm-2"><strong>Conservation status</strong></div>
+
+                <div class="col-sm-10">
+                    <div class="col-sm-4" ng-repeat="status in listCtrl.conservationStatuses">
+                        <a href="${grailsApplication.config.collectory.base.url}/public/showDataResource/{{listCtrl.statusRegions[status.region].id}}"
+                           title="Threatened Species Codes - details" target="_blank">
+                            <div class="status"
+                                 ng-class="listCtrl.getColourForStatus(status.status)">{{listCtrl.statusRegions[status.region].abbrev | default:'IUCN'}}</div>
+                            {{status.rawStatus}}
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    %{--for bio status--}%
+    <div class="panel panel-default" ng-show="listCtrl.bioStatuses.length > 0">
+        <a name="{{listCtrl.readonly ? 'view_' : 'edit_'}}bioStatus"></a>
+
+        <div class="panel-body">
+            <div class="row">
+                <div class="col-sm-2"><strong>Bio Status</strong></div>
+
+                <div class="col-sm-10">
+                    <ul>
+                        <li ng-repeat="status in listCtrl.bioStatuses">
+                            {{status.key}} : {{status.value}}
```

Comment:
We'll need to make sure the text from the list is formatted correctly: replace underscores with spaces, change to sentence case, etc. Have a look at the "capitalize" filter in the utils.js file for an example of how to do this with a filter. Create a new 'formatText' filter so we can use {{ status.key | formatText }} or similar.